uaclient/security.py

```diff
@@ -822,6 +822,37 @@ def _prompt_for_attach(cfg: UAConfig) -> bool:
     return True
 
 
+def _prompt_for_enable(cfg: UAConfig, service: str) -> bool:
+    """Prompt for enable a ua service.
+
+    :return: True if enable performed.
+    """
+    import argparse
+    from uaclient import cli
+
+    print(status.MESSAGE_SECURITY_SERVICE_DISABLED.format(service=service))
+    choice = util.prompt_choices(
+        "Choose: [E]nable {} [C]ancel".format(service),
+        valid_choices=["e", "c"],
+    )
+
+    if choice == "c":
+        return False
+    elif choice == "e":
```

Comment:
Could consolidate this section logically dropping the if choice == "c" because we fall through and return False at end
```suggestion
    if choice == "e":
```